tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -53,11 +53,11 @@ arp/test_unknown_mac.py:
 
 arp/test_wr_arp.py:
   skip:
-    reason: "Warm reboot is broken on dualtor topology. Device fails to recover by sanity check. Skipping for now."
+    reason: "Warm reboot is broken on dualtor topology. Device fails to recover by sanity check. Skipping for now. Not yet supported in TH5"
+    conditions_logical_operator: or
     conditions:
-      - "'dualtor' in topo_name"
-      - https://github.com/sonic-net/sonic-buildimage/issues/16502
-
+      - "https://github.com/sonic-net/sonic-buildimage/issues/16502 and 'dualtor' in topo_name"
+      - "asic_gen in ['th5']"
```

Comment:
will be better to move to per topology condition. i will get the topologies in, and let's switch to them.

Reply:
@r12f I could change to standalone topo for now and we could address additional topos as they come up. What do you think?

Reply:
Changed from th5 to standalone topos.